internal/api/dto/price.go

```diff
@@ -23,7 +23,7 @@ type CreatePriceRequest struct {
 	Type               types.PriceType          `json:"type" validate:"required"`
 	PriceUnitType      types.PriceUnitType      `json:"price_unit_type" validate:"required"`
 	BillingPeriod      types.BillingPeriod      `json:"billing_period" validate:"required"`
-	BillingPeriodCount int                      `json:"billing_period_count" validate:"required,min=1"`
+	BillingPeriodCount int                      `json:"billing_period_count"`
```

Comment:
we can give default value here itself 
Handling in Validate methods is still okay and should be there so no need to change that.
Lets just update the tag to set default values here itself.

```suggestion
	BillingPeriodCount int                      `json:"billing_period_count" default:"1"`
```

Reply:
Need to do this for other structs as well